### PR TITLE
Install Monstache (mongo -> elasticsearch sync program) to buildpack

### PR DESCRIPTION
**Pull request by Mainstay-Noah-Huppert**
ALTA-635

Change log:

- Fixed AWS cli Python 2 install error
- Made buildpack download the Monstache binary into marshall's `monstache/` directory

### CODE REVIEW

File: bin/compile [hunk 1]
```diff
@@ -116,6 +126,31 @@ tar xf "${frontend_asset_file}" -C "${frontend_dir}"
...
+readonly monstache_dl_url="https://github.com/rwynn/monstache/releases/download/v6.7.5/monstache-e9c8b73.zip"
+readonly monstache_dl_sha256="ea0d6fdd2f27ecea9b6fd86c308cae6089c9cfa011cf528b937cdc8de9edea59"
+readonly monstache_dl_path="$monstache_dir/monstache.zip"
+
+readonly monstache_unzipped_dir="$monstache_dir/build"
+readonly monstache_unzipped_bin="$monstache_unzipped_dir/linux-amd64/monstache"
+readonly monstache_unzipped_bin_sha256_file="$monstache_unzipped_dir/linux-amd64/sha256.txt"
+readonly monstache_bin_install_path="$monstache_dir/monstache"
+
+curl -L --create-dirs -o "$monstache_dl_path" "$monstache_dl_url"
```
Comment (sbdchd) [inline bin/compile:142]: do we need to `mkdir -p` for these dirs?

Comment (Mainstay-Noah-Huppert) [reply bin/compile:142]: I believe `curl`'s `--create-dirs` option takes care of this for us:
> --create-dirs
>               When used in conjunction with the -o, --output option,
>               curl will create the necessary local directory hierarchy
>               as needed. This option creates the directories mentioned
>               with the -o, --output option, nothing else. If the
>               --output file name uses no directory, or if the
>               directories it mentions already exist, no directories will
>               be created.

I have [a review app](https://dashboard.heroku.com/apps/marshall-review-br-noah-alta-6) deployed with this PR's branch and it seems to work.

**Review by chdsbd — APPROVED** [review]

**Review by sbdchd — APPROVED** [review]
🚢

**Comment by Mainstay-Noah-Huppert** [comment]
Would you say this PR warrants a minor version tag bump to 4.3? @chdsbd @sbdchd ?

**Comment by chdsbd** [comment]
@Mainstay-Noah-Huppert Sounds good to me. I think we usually reference the tag explicitly in the Heroku config, so we should update that on staging too when we're ready

**Comment by Mainstay-Noah-Huppert** [comment]
> @Mainstay-Noah-Huppert Sounds good to me. I think we usually reference the tag explicitly in the Heroku config, so we should update that on staging too when we're ready

Okay sounds good. I'll have a marshall PR to update `app.json` and then we can also update the heroku buildpack settings on the dashboard for staging and then production after a few days.

**Pull request merged**